extensions/widgets/treeview/treeview.lcb

```diff
@@ -104,6 +104,21 @@ The arrayData is the data currently being displayed by the tree view widget.
 property arrayData 					get getArrayData 				set setArrayData
 metadata arrayData.label is "Array data"
 
+/**
+Syntax: set the foldState of <widget> to <pFoldState>
+Syntax: get the foldState of <widget>
+Summary: The fold state of the array being displayed by the widget
+
+Parameters:
+pFoldState (array): The fold state data.
```

Comment:
@bwmilby could you describe the array a bit here

Reply:
Will do.